lucid/launcher.py

```diff
@@ -162,6 +162,7 @@ def launch(beamline, *, toolbar=None, row_group_key="location_group",
 
     # Silence the logger from pyPDB.dbd.yacc
     logging.getLogger("pyPDB.dbd.yacc").setLevel(logging.WARNING)
+    logging.getLogger("ophyd").setLevel(logging.WARNING)
```

Comment:
Sneaky

Reply:
I forgot this here... should I remove it or keep it until we fix the logger?

Reply:
We should leave it because it improves the user experience